### DISCUSSION TITLE
Detecting on-hold/transfer music during phone calls

### DISCUSSION THREAD

**Discussion by netw0rkf10w** [deepgram/community#702] [Feature Requests]
During a phone call, one may put the other person on hold or may transfer the call to another person, and most of the time there some music playing. Can the current API detect this already?

Thank you in advance for your reply!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by netw0rkf10w** [comment]
Any ideas on this please? @jpvajda @RyanChimienti

**Comment by jpvajda** [comment]
@netw0rkf10w Our [nova-2 Phone Call Model](https://developers.deepgram.com/docs/models-languages-overview#nova-2) model should be able to filter out that music as background noise and ignore it. 

It best to try it out though, I'd suggest using our [API Playground](https://playground.deepgram.com/?endpoint=listen&smart_format=true&language=en&model=nova-2-phonecall) to run some sample audio requests and see how it handles it.
